bower.json

```
@@ -1,23 +1,24 @@
 {
   "name": "adapt-mediaWithBenefits",
-  "version": "0.0.1",
-  "framework": ">=3",
+  "version": "0.0.2",
```

We've done major version bumps for all the other v5 compatible releases
```suggestion
  "version": "1.0.0",
```